src/layout/section/baseSection/BaseSection.style.tsx

```diff
@@ -0,0 +1,27 @@
+import styled from 'styled-components'
+
+import { componentSizes, horizontalSpace, responsiveBreakpoints } from '../../../_utils/branding'
+
+export const StyledBaseSection = styled.div`
```

Comment:
should base "section" be a `section` tag?

Reply:
Not in the scope of this PR I guess, but I don't have a strong feeling about this